.github/workflows/continuous-integration.yaml

```diff
@@ -1,7 +1,7 @@
 name: Continuous Integration
 
 on:
-  push:
+  pull_request:
```

Comment:
Can you please keep this file aligned with the configuration from my other repositories, e.g. https://github.com/ricoberger/echoserver/blob/main/.github/workflows/continuous-integration.yaml

Reply:
Can we use push & pull_request?
Otherwise ci does not run on PR?